app/assets/javascripts/services/alerts_center_service.js

```diff
@@ -393,11 +393,30 @@ function alertsCenterService(API, $q, $timeout, $document, $modal) {
       .then(_this.updateExistingUsers)
       .then(_this.updateProviders)
       .then(_this.updateTags)
+      .then(_this.updateHostIcons)
       .then(function() {
         return _this.getAlertsData(limit, offset, filters, sortField, sortAscending);
       });
   };
 
+  _this.updateHostIcons = function(response) {
+    var deferred = $q.defer();
+    $http.get('/alerts_list/class_icons').then(function(response) {
+      console.log(response.data);
+      _this.hostIcons = response.data;
+      deferred.resolve(response);
+    });
+    return deferred.promise;
+  };
```

Comment:
... I guess you forgot to remove the original version? I'm seeing `updateHostIcons` twice now.